client/src/legacy/entwine/LeftAndMain_Subsites.js

```diff
@@ -0,0 +1,190 @@
+/* jslint browser: true, nomen: true */
+/* global window, jQuery */
+import React from 'react';
+import ReactDom from 'react-dom';
+import { loadComponent } from 'lib/Injector';
+import createEvent from 'legacy/createEvent';
+import changeActiveSubsite from 'lib/changeActiveSubsite';
+
+jQuery.entwine('ss', ($) => {
+  $('#SubsitesSelect').entwine({
+    ModalNode: null,
+    onadd() {
+      // Storage has updated - another tab has changed the subsite info
+      window.addEventListener('storage', (storageEvent) => {
+        if (storageEvent.key === 'subsiteInfo') {
+          const subsiteChangeEvent = createEvent('subsitechange', { subsiteInfo: storageEvent.newValue });
+          window.dispatchEvent(subsiteChangeEvent);
+        }
+      }, false);
+
+      window.addEventListener('subsitechange', (subsiteChangeEvent) => {
+        const subsiteNotice = this.getModalNode();
+        const subsiteInfo = JSON.parse(subsiteChangeEvent.subsiteInfo);
+        if (subsiteNotice) {
+          ReactDom.unmountComponentAtNode(subsiteNotice);
+        }
+        if (subsiteInfo.subsiteID !== this.val()) {
+          this.showReactiveNotice(subsiteInfo.subsiteName);
+        }
+      }, false);
+
+      // Dropdown change trigger
+      this.on('change', () => {
+        const subsiteID = this.val();
+        window.location.search = $.query.set('SubsiteID', subsiteID);
+      });
+
+      // We need to set when a page loads, as it may be e.g. the refresh of a currently blocked tab.
+      this.storeSubsiteInfo();
+    },
+    storeSubsiteInfo() {
+      const subsiteID = this.val();
+      const subsiteInfo = {
+        subsiteID,
+        subsiteName: $(`[value="${subsiteID}"]`, this).text()
+      };
+      window.localStorage.setItem('subsiteInfo', JSON.stringify(subsiteInfo));
+    },
+    showReactiveNotice(newSubsiteName) {
+      const { document } = window;
+      // React business
+      const modalContainer = this.getModalNode() || document.createElement('div');
+      document.body.appendChild(modalContainer);
+      const ChangeAlert = loadComponent('SubsiteChangeAlert');
```

Comment:
should be const SubsiteChangeAlert so that the jsx below has the correct component name?